server/lib/src/entry.rs

```diff
@@ -2471,6 +2473,16 @@ impl<VALID, STATE> Entry<VALID, STATE> {
             .and_then(|vs| vs.to_jws_key_es256_single())
     }
 
+    pub fn get_ava_single_eckey_private(&self, attr: &str) -> Option<EcKey<Private>> {
+        self.attrs
+            .get(attr)
+            .and_then(|vs| vs.to_eckey_private_single())
+    }
```

Comment:
You know what :)

Reply:
dang I'll develop a tool that checks for spaces after functions :p